### DISCUSSION TITLE
Modifying front-end code using docker or backend runtime without any changes

### DISCUSSION THREAD

**Discussion by Bc-Aqr** [apache/superset#25887] [Q&A / Help]
**update  superset/superset-frontend/src/features/home/Menu.tsx**

When I run the front-end npm run server alone, I can see the modified page, but when I am deployed with docker or the back-end is running locally, how the front-end is modified has not changed, why?

**The following actions have been tried, but they still do not work, such as:**
1. Use docker compose -no-cache
2.Clear browser cache
3.Manually delete the locally packaged dist file again and perform docker deployment

**Comment by DarwinDennis003** [comment]
Is your docker image custom build  or from superset image from docker hub .Cross check that..

**Reply by Bc-Aqr** [reply]
I am very glad to get your reply，The docker image uses the native docker hub

**Reply by DarwinDennis003** [reply]
Since the docker image is native superset image your changes wont be reflected .You should build your own docker image.
